internal/compiler/opcodes.go

```diff
@@ -68,6 +68,9 @@ const (
 	// Multi-index concatenation
 	MultiIndex // num
 
+	// Multi-value concatenation
+	ConcatN
```

Comment:
Let's consider just making `Concat` always take an argument (numValues), assuming that doesn't slow down the concat-two case significantly. Could you please add a benchmark to interp_test.go `BenchmarkConcatTwo` that benchmarks this case, and post before/after results?

Reply:
You'll also need to add ConcatN (or Concat if that's what we end up calling it) to the disassembler, so it consumes and displays the numValues argument.

Reply:
The original concat with 2 arguments seems to be at least twice as fast than using the new one:

`
// ConcatN
  BenchmarkConcatTwo-16               	 8443903	       139.6 ns/op
  BenchmarkConcatTwo-16               	 8600488	       137.8 ns/op
  BenchmarkConcatTwo-16               	 8743024	       141.4 ns/op
  BenchmarkConcatTwo-16               	 8841157	               138.4 ns/op
  BenchmarkConcatTwo-16               	 8648384	       139.0 ns/op
---
// Concat
 BenchmarkConcatTwo-16               	15598926	        74.76 ns/op
 BenchmarkConcatTwo-16               	16030410	        76.17 ns/op
 BenchmarkConcatTwo-16               	15828648	        74.84 ns/op
 BenchmarkConcatTwo-16               	16167350	        76.84 ns/op
 BenchmarkConcatTwo-16               	15776442	        75.88 ns/op
`

Reply:
Dropping to 98 when using strings.Builder in the VM interpreter:

`
BenchmarkConcatTwo-16               	12588488	        97.84 ns/op
BenchmarkConcatTwo-16               	12410306	        98.79 ns/op
BenchmarkConcatTwo-16               	12267706	        98.20 ns/op
BenchmarkConcatTwo-16               	12534445	        98.70 ns/op
BenchmarkConcatTwo-16               	12488019	        98.12 ns/op
`